pkg/cli/cvd.go

```diff
@@ -726,30 +692,67 @@ func envVar(name string) (string, error) {
 	return os.Getenv(name), nil
 }
 
-func uploadFiles(client hoclient.HostOrchestratorClient, uploadDir string, names []string, statePrinter *statePrinter) error {
-	extractOps := []string{}
-	for _, name := range names {
-		state := fmt.Sprintf("Uploading %q", filepath.Base(name))
+func uploadFiles(client hoclient.HostOrchestratorClient, filenames []string, statePrinter *statePrinter) (string, error) {
```

Comment:
nit: this function is doing more than just uploading files, consider renaming to `uploadFilesAndcreateImageDir`